src/test/java/com/github/emilshina/data/CsvReader.java

```diff
@@ -0,0 +1,36 @@
+package com.github.emilshina.data;
+
+import com.github.emilshina.utils.ReflectionUtils;
+import io.vavr.control.Try;
+import lombok.extern.slf4j.Slf4j;
+import one.util.streamex.StreamEx;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static java.lang.ClassLoader.getSystemResource;
+
+/**
+ * Class for reading data from csv files.
+ */
+
+@Slf4j
+public class CsvReader implements DataReader {
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public <T> T[] readFrom(final String dataSource, final Class<T> entityClass) {
+        return (T[]) Try.withResources(() -> Files.lines(Paths.get(getSystemResource(dataSource).getPath())))
+                .of(stringStream -> StreamEx.of(stringStream)
+                        .skip(1)
+                        .map(line -> line.split(";"))
```

Comment:
do you want to move separator into properties?

Reply:
Will do it in another branch, because the file with properties is created in branch 4-webdriwer-tests. I want to avoid an additional merge.